client/src/components/DragAndDrop/dNdComponents/itemList.jsx

```diff
@@ -0,0 +1,141 @@
+import { useState, useRef } from "react";
+import { Droppable, Draggable } from "react-beautiful-dnd";
+import ColumnItem from "./item";
+import dndStyles from '../assets/dndStyles'
+import { Box, Grid, Typography, Button, Paper, Avatar, TextField } from '@material-ui/core';
+
+
+export const Wrapper = ({ classes, isDraggingFrom, isDraggingOver, children}) => <Grid item className={isDraggingOver ?
```

Comment:
What does it wrap? I would recommend naming this component more descriptively - especially because we're exporting it (implying it could be used anywhere in the project)👍🏼 Or maybe we don't really want to export it?